solidity/ecdsa/test/WalletRegistry.WalletCreation.test.ts

```diff
@@ -1835,7 +1845,8 @@ describe("WalletRegistry - Wallet Creation", async () => {
 
               await mineBlocksTo(startBlock + dkgTimeout - 1)
 
-              let submitter
+              let members: number[]
```

Comment:
This guy is not used anymore. And also the 'misbehavedIds' has not type, we can declare it as number[]

Reply:
See https://github.com/keep-network/keep-core/pull/2868/commits/c1d223c805be6558e25c7ae997187f54c409cb34 ser.